modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -805,21 +810,23 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
      *  @param swapRB flag which indicates that swap first and last channels
      *  in 3-channel image is necessary.
      *  @param crop flag which indicates whether image will be cropped after resize or not
+     *  @param type Type of output blob.
      *  @details if @p crop is true, input image is resized so one side after resize is equal to corresponding
      *  dimension in @p size and another one is equal or larger. Then, crop from the center is performed.
      *  If @p crop is false, direct resize without cropping and preserving aspect ratio is performed.
      *  @returns 4-dimensional Mat with NCHW dimensions order.
      */
     CV_EXPORTS_W Mat blobFromImage(InputArray image, double scalefactor=1.0, const Size& size = Size(),
-                                   const Scalar& mean = Scalar(), bool swapRB=true, bool crop=true);
+                                   const Scalar& mean = Scalar(), bool swapRB=true, bool crop=true,
+                                   int type=CV_32F);
```

Comment:
`CV_32F` is data depth.
OpenCV's "type" contains channels information too (`CV_32FC1`).

`type` => `depth` anywhere in this patch?

Reply:
`dtype` name itself seems ambiguous to me:
- destination type
- depth of type

So as you wish.